### PR TITLE
Rotate map icon instead of view

### PR DESCRIPTION
**Pull request by atomgomba**
Follow up to #3355. Instead of rotating the map, this PR makes only the icon rotate according to heading.

### CODE REVIEW

**Comment by nerdCopter** [comment]
@haslinghuis i cannot test for review (i do not have GPS). i see the code is a simple change.

**Comment by haslinghuis** [comment]
We need some little adjustments in gps.js. Just found out the formula for degrees to radians was wrong and we send the gpsHeading instead of the imuHeading. I'll raise a PR to adjust that now.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
icon rotates perfectly!  Thank you!

**Pull request merged**
